app/src/main/java/com/microsoft/research/karya/ui/dashboard/DashboardFragment.kt

```diff
@@ -212,14 +212,6 @@ class DashboardFragment : SessionFragment(R.layout.fragment_dashboard) {
             (binding.tasksRv.adapter as TaskListAdapter).updateList(taskInfoData)
         }
 
-        // Check if worker is initialised in viewmodel
-        if (viewModel.workerAccessCode.value.isNotEmpty()) {
```

Comment:
Dont remove the code. We just have to make sure it runs once. Make a global Boolean or a shared preference to make sure it runs only once